setup.cfg

```diff
@@ -28,9 +28,9 @@ python_files = tests.py test_*.py tests_*.py *_tests.py __init__.py
 #   http://nedbatchelder.com/blog/200711/rethrowing_exceptions_in_python.html
 #   It's a little unusual, but we have good reasons for doing so, so we disable
 #   this rule.
-# E305,E402,E722,E731,E741,E743,W503: errors and warnings added since pep8/pycodestyle
+# E305,E402,E722,E731,E741,E743,W503,W504: errors and warnings added since pep8/pycodestyle
 # 1.5.7 that we haven't cleaned up yet
-ignore=E265,E266,E305,E402,E501,E722,E731,E741,E743,W503,W602
+ignore=E265,E266,E305,E402,E501,E722,E731,E741,E743,W503,W504,W602
```

Comment:
could you clarify why this change is necessary?  
I don't think you're using it in this PR

Reply:
W504 is newly added by pycodestyle. We have code that violates it, that I'm not cleaning up right now.